in_text/text_data_release.yml

```
@@ -124,7 +124,37 @@ funding-credits: >-
   This research used resources of the Core Science Analytics and Synthesis Advanced Research Computing program at the U.S. Geological Survey.
 
 process-description: >-
-  [A description of processing]
+  This dataset was generated using two modeling frameworks: 1) a lake hydrodynamic model that uses inputs of lake-specific properties and local 
+  meteorology to estimate water temperature, and 2) [LSTM description from Andy]. Lake-specific properties and temperature obvservations were 
+  aggregated from a variety of sources and cooperators for lakes across the US Midwest. Meteorological inputs for the contemporary period (1981-
+  2000) used by both modeling frameworks were downloaded from the North American Land Data Assimilation System (NLDAS). Meteorological inputs for 
```

@padilla410 when you mentioned referencing the citation in our process description, were you thinking something like this?

```suggestion
  2000) used by both modeling frameworks were downloaded from the North American Land Data Assimilation System (NLDAS, Mitchell et al 2004). Meteorological inputs for 
```

---

Yes, perfect. I'll add the suggested code snippet into our running issue of references